Week3/homework/step2-3.js

```diff
@@ -4,9 +4,15 @@
 function repeatStringNumTimesWithFor(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
+  const repeatArr = [];
+  if (num <= 0) {
```

Comment:
This extra check is unneeded because the `for` loop body will also not be executed if `num <= 0` if you leave out this test.